### PR TITLE
update stage 3 to most recent

### PR DESCRIPTION
**Pull request by codehag**

### CODE REVIEW

**Comment by codehag** [comment]
cc @smorimoto @rkirsling

**Comment by smorimoto** [comment]
I will check that! Thanks for your work, as always 🙂

**Comment by codehag** [comment]
Thanks @smorimoto -- sorry it is a bit late. The translations are not there yet, but I want to make sure I am not overwriting your work.

**Comment by smorimoto** [comment]
No worries at all, take your time :)

**Review by smorimoto — APPROVED** [review]

File: _data/ja/stage3.yml [hunk 1]
```diff
@@ -279,3 +279,84 @@
...
   champions:
     - Jordan Harband
+  has_specification: true
+- id: proposal-class-static-block
+  title: Class Static Block
+  authors:
+    - Ron Buckton
+  champions:
+    - Ron Buckton
+  description: Class static blocks provide a mechanism to perform additional static initialization during class definition evaluation. This is not intended as a replacement for public fields, as they provide useful information for static analysis tools and are a valid target for decorators. Rather, this is intended to augment existing use cases and enable new use cases not currently handled by that proposal.
```
Comment (smorimoto) [inline _data/ja/stage3.yml]:
```suggestion
  description: Class static blockは、クラス定義の評価中に追加の静的初期化を実行する仕組みを提供します。これは、静的解析ツールに役立つ情報を提供し、デコレーターのための妥当なターゲットであるため、パブリックフィールドの置き換えを意図したものではありません。むしろ、これは既存のユースケースを補強し、このプロポーザルで現在処理されていない新しいユースケースを実現できるようにすることを目的としています。
```

File: _data/ja/stage3.yml [hunk 2]
```diff
@@ -279,3 +279,84 @@
...
+        https://github.com/tc39/notes/blob/master/meetings/2021-03/mar-9.md#class-static-initialization-blocks
+- id: proposal-error-cause
+  title: Error Cause
+  authors:
+    - Chengzhong Wu
+  champions:
+    - Ron Buckton
+    - Chengzhong Wu
+    - Hemanth HM
+  description: Errors will be constructed to represent runtime abnormalities. To help unexpected behavior diagnosis, errors need to be augmented with contextual information like error messages, error instance properties to explain what happened at the time.  If the error were thrown from deep internal methods, the thrown error may not be straightforward to be easily conducted without proper exception design pattern. This proposal introduces a convenient way to forward the cause.
```
Comment (smorimoto) [inline _data/ja/stage3.yml]:
```suggestion
  description: エラーは、実行時の異常を表現するために作られます。予期しない動作の診断を容易にするためは、エラーメッセージやエラーインスタンスのプロパティなど、その時点で何が起こったかを明らかにする状況情報をエラーに追加する必要があります。エラーが内部の深いメソッドからスローされた場合、スローされたエラーは、適切な例外設計パターンなしに簡単に処理できるとは限りません。このプロポーザルは原因を伝達するための便利な手段を導入します。
```

File: _data/ja/stage3.yml [hunk 3]
```diff
@@ -279,3 +279,84 @@
...
+  title: Temporal
+  authors:
+    - Maggie Pint
+    - Matt Johnson
+    - Philipp Dunkel
+  champions:
+    - Maggie Pint
+    - Matt Johnson
+    - Philipp Dunkel
+  description: Date has been a long-standing pain point in ECMAScript. This proposes Temporal, a global Object that acts as a top-level namespace (like Math), that brings a modern date/time API to the ECMAScript language. For a detailed breakdown of motivations, see  the proposal readme.
```
Comment (smorimoto) [inline _data/ja/stage3.yml]:
```suggestion
  description: ECMAScriptでは、Dateが長年の悩みの種でした。そこで提案されたのがTemporalであり、これは(Mathのような)トップレベルの名前空間として振る舞い、ECMAScriptに最新のdate/time APIをもたらすグローバルオブジェクトです。動機の詳しい説明については、プロポーザルのREADMEをご覧ください。
```

**Comment by smorimoto** [comment]
I just added new Japanese translations for stage3.

**Pull request merged**